src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs

```diff
@@ -6,11 +6,15 @@ public sealed class ChocolateyPackage : Resource
     {
         public string Package { get; set; }
         public PackageState Ensure { get; set; }
+        public bool PreRelease { get; set; }
+        public bool IgnoreChecksum { get; set; }
+        public string PackageParameters { get; set; }
```

Comment:
Could we make this nullable?

Reply:
the string or all 3?

Reply:
Sorry, I missed this comment. I meant `PackageParameters`.